### DISCUSSION TITLE
Image Load Error: Failed to load resource: the server responded with a status of 504 ()

### DISCUSSION THREAD

**Discussion by mayankzalavadia** [vercel/next.js#23763] [Help]
Hello All,
Hope you all are fine.

We are developing an eCommerce website with Magento and nextjs and we have faced image-related issues. So, in that case, I need your help.
Actually, we have face 504 error issues in images after the update to the **latest** nextjs version.

So, can anyone please suggest to me if you have any solution for that type of errors?

**FYI:** We use the Nginx server

So, let us know if is there any Nginx configuration required for **_next/image**.

Thank you in advance.

Regard,
Mayank

**Comment by kristianstroka** [comment]
I have same issue with latest canary verison

**Reply by mayankzalavadia** [reply]
@kristianstrokaa

I have tried a lot but not gonna succeed. After that, I have created this discussion for help but unfortunately, no one can answer. I feel really helpless not even the core team answer. I think the nextjs core team has to help us to improve their community knowledge and try to build a strong community.

I hope anyone will help us from the community or core team to solve this issue.

**Reply by mayankzalavadia** [reply]
@timneutkens Can you please help us with this issue?

**Reply by kristianstroka** [reply]
I found issue on my environment, I had node in version 10, after update to 14LTS it works as expected

**Reply by mayankzalavadia** [reply]
Thank you @kristianstroka I have updated the node version and it's working.

Thank you so much. Let's connect on Twitter. My Twitter handle is : [mayankzalavadia](https://twitter.com/mayankzalavadia)
